reactor-core/src/main/java/reactor/core/publisher/FluxProcessor.java

```diff
@@ -226,7 +226,12 @@ public Context currentContext() {
 	 * </ul>
 	 *
 	 * @return a serializing {@link FluxSink}
-	 * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+	 * @deprecated To be removed in 3.5, prefer clear cut usage of {@link Sinks}:
+	 * <pre>
+	 *     final Sinks.Many<String> sink = Sinks.many().replay().latest();
```

Comment:
@simonbasle I need your opinion on this one

Reply:
since there is no "flavor" to the `sink` method itself (it is driven by the concrete implementation on which it is called), I'd remove the mention of any specific full builder and just say `usage of {@code Sinks} through the {@link Sinks#many()} spec` (and remove the `pre` block)